libraries/lib-effects/PerTrackEffect.cpp

```diff
@@ -31,8 +31,6 @@
 #include "WaveTrackSink.h"
 #include "WideSampleSource.h"
 
-AudioGraph::Sink::~Sink() = default;
-
```

Comment:
Tell me what warning this fixes

Reply:
`warning C4273: 'AudioGraph::Sink::~Sink': inconsistent dll linkage `

Reply:
I see, it was a duplicate definition of `~Sink` which is still defined elsewhere.